temporal-sdk/src/main/java/io/temporal/workflow/NexusOperationCancellationType.java

```diff
@@ -0,0 +1,29 @@
+package io.temporal.workflow;
+
+import io.temporal.failure.CanceledFailure;
+
+/**
+ * Defines behavior of the parent workflow when {@link CancellationScope} that wraps Nexus operation
+ * is canceled. The result of the cancellation independently of the type is a {@link
+ * CanceledFailure} thrown from the Nexus operation method.
+ */
+public enum NexusOperationCancellationType {
```

Comment:
Are we releasing this as GA or will it be Pre-release/Public preview?

Reply:
Shouldn't it be Pre-release/Public preview, we talked about it during the nexus crew stand up ~2 weeks ago?

Reply:
We did end up marking it experimental for Go, lets mark it experimental here as well